includes/class-donations.php

```diff
@@ -801,35 +801,59 @@ public static function checkout_create_order_line_item( $item, $cart_item_key, $
 	public static function create_donation_page() {
 		$revenue_model = 'donations';
 
-		$intro           = esc_html__( 'With the support of readers like you, we provide thoughtfully researched articles for a more informed and connected community. This is your chance to support credible, community-based, public-service journalism. Please join us!', 'newspack' );
-		$content_heading = esc_html__( 'Donation', 'newspack' );
-		$content         = esc_html__( "Edit and add to this content to tell your publication's story and explain the benefits of becoming a member. This is a good place to mention any special member privileges, let people know that donations are tax-deductible, or provide any legal information.", 'newspack' );
-
+		$title   = esc_html__( 'Support our publication', 'newspack' );
+		$intro   = esc_html__( 'With the support of readers like you, we provide thoughtfully researched articles for a more informed and connected community. This is your chance to support credible, community-based, public-service journalism. Please join us!', 'newspack' );
+		$content = esc_html__( "Edit and add to this content to tell your publication's story and explain the benefits of becoming a member. This is a good place to mention any special member privileges, let people know that donations are tax-deductible, or provide any legal information.", 'newspack' );
+
+		$container_block       = '
+			<!-- wp:cover {"url":"https://newspack.com/wp-content/uploads/2024/05/pexels-brotin-biswas-158640-518543-scaled.jpg","dimRatio":60,"overlayColor":"black","isUserOverlayColor":true,"minHeight":80,"minHeightUnit":"vh","align":"full","style":{"spacing":{"padding":{"top":"var:preset|spacing|80","bottom":"var:preset|spacing|80","left":"var:preset|spacing|80","right":"var:preset|spacing|80"}}},"layout":{"type":"constrained"}} -->
```

Comment:
This assumes that the `https://newspack.com/wp-content/uploads/2024/05/pexels-brotin-biswas-158640-518543-scaled.jpg` file will always be available, which is not a given.

Reply:
I don't think this block pattern should rely on any externally hosted image. The user may be happy with the image (so they don't change it), and then later it's removed from the external host for any reason. The page would suddenly be broken. The image should be downloaded first, and used from the site's media library.

Reply:
Ok I moved the image to `/includes/`. I also changed it to make sure a publisher will replace it!

https://github.com/Automattic/newspack-plugin/pull/3109/commits/d662d423e5aeca41204db579a99bdb450370c29e

Reply:
Awesome, thank you @adekbadek.

 > WDYT about renaming the image file to something like "placeholder-image"

Done in https://github.com/Automattic/newspack-plugin/pull/3109/commits/8c1236d95615f4e99b48c06ef285a1e9ceb7af99

> These instances should be updated to use the placeholder image introduced in this PR

I'll create a new issue for the `newspack-blocks` patterns 👍

> If a pattern was inserted, and then updated in this repository, will it be updated on a post that used the pattern.

No, the pattern will be the old one.